AutomatticTracks/build.gradle

```diff
@@ -38,16 +38,26 @@ android {
     }
 }
 
-
-publish {
-    userOrg = 'automattic'
-    groupId = 'com.automattic'
-    uploadName = 'tracks'
-    artifactId = 'tracks'
-    desc = 'Android client for Nosara tracks (event tracking and analytics)'
-    publishVersion = android.defaultConfig.versionName
-    licences = ['MIT']
-    website = 'https://github.com/Automattic/Automattic-Tracks-Android'
-    dryRun = 'false'
-    autoPublish = 'true'
+afterEvaluate {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                artifactId 'Automattic-Tracks-Android'
+                artifact("$buildDir/outputs/aar/AutomatticTracks-debug.aar")
+                description 'Android client for Nosara tracks (event tracking and analytics)'
+                groupId 'com.github.Automattic'
+                pom {
+                    licenses {
+                        license {
+                            name = 'GNU General Public License v2.0'
+                            url = 'https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html'
+                        }
+                    }
+                }
+                if (project.hasProperty("localDistribution") && project.getProperty("localDistribution").toBoolean()) {
+                    version = "LOCAL-SNAPSHOT"
+                }
```

Comment:
I think we should remove everything related to local maven from the changeset. I'd be happy to assist with composite builds next week, so we don't need it.